templates/log-book-report/templates/main.tpl

```diff
@@ -0,0 +1,193 @@
+{{define "received-header"}}
+    {{range $item := (getSlice "#,Manufacturer,Model,VIN,DateReceived,Source")}}
+        <table-cell background-color="#cfcfcb" border-width-bottom="0.5" border-width-top="0.5">
+            <paragraph>
+                <text-chunk font="exo-bold" font-size="11">{{$item}}</text-chunk>
+            </paragraph>
+        </table-cell>
+    {{end}}
+    {{range $i, $item := (getSlice "#,Manufacturer,Model,VIN,DateReceived,Source")}}
+        <table-cell border-width-bottom="0.5" border-width-top="0.5">
+            <paragraph>
+                {{if eq $i 1}}
+                    <text-chunk font="exo-regular" font-size="11">Description of model</text-chunk>
+                {{else}}
+                    <text-chunk font="exo-regular" font-size="11"></text-chunk>
+                {{end}}
+            </paragraph>
+        </table-cell>
+    {{end}}
+{{end}}
+
+{{define "sent-header"}}
+    {{range $columName := (getSlice "#,Date Sent,Buyer Name,Buyer Address")}}
```

Comment:
Same here. Maybe create a variable for `#,Date Sent,Buyer Name,Buyer Address` as it is used again in the next block.